src/ralph/accounts/models.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+from dj.choices import (
+    Country,
+    Gender
```

Comment:
in whole file add trailing commas at the end of multiline import (here) and fields declaration :)